### ISSUE TITLE
Title weirdness

### ISSUE THREAD

**Issue by YouveGotMeowxy** [wallabag/wallabagger#378]
Version: 1.16.0 on the latest Brave.

When clicking the icon to save the page, the proper title initially shows, and then changes to a non-descriptive title instead.

![2024-01-09_18-23-06](https://github.com/wallabag/wallabagger/assets/39491297/790790b2-82ff-47ba-9a86-c20b68e1e3f1)

example URL:

https://one.google.com/u/0/dwr/setup_profile?attribute_type=0&hl=en

**Comment by Simounet** [comment]
Seems like it's not a public page. Did you added this domain to your extension option to be fetch from the browser?

**Comment by YouveGotMeowxy** [comment]
> Seems like it's not a public page. Did you added this domain to your extension option to be fetch from the browser?

It's just Google. If you login with your usual Google account (Gmail, etc.) you'll get to the page.

**Comment by Simounet** [comment]
Yes but your wallabag instance is not logged.

**Comment by YouveGotMeowxy** [comment]
Sorry, I don't understand; do you mean you want to see the logs from my Wallabag server?

**Comment by YouveGotMeowxy** [comment]
Or do you mean to add google.com to this field?

![image](https://github.com/wallabag/wallabagger/assets/39491297/5f641cef-b03b-44a8-98b6-010afd7f778c)

**Comment by Simounet** [comment]
Nope. I mean that it is not your browser, logged in, that send the content to wallabag.
1. wallabagger send the URL to the server
2. the server fetch the URL's content
So the content should publicly available. If you want wallabagger to send the content to wallabag, you need to add the URL into this box.

**Comment by HolgerAusB** [comment]
Most paywalled or login-needing pages can be fetched with wallabager's 'content fetching from browser' option. So yes, put google.com or one.google.com to that field or activate this option as default for all fetches.

**Comment by HolgerAusB** [comment]
If you don't activated this, wallabagger only sends the URL to wallabag while wallabag is not able to login to google.

**Comment by YouveGotMeowxy** [comment]
Ohhhh ok. I never knew how that field worked. I'll give it a try.

Update:
Hmm, I added "google.com" (no quotes) to the field, loaded up a new private window, went to the page, it asked me to login w/my google acct, I did, clicked the wbagger icon, and the same thing happens.

**Comment by YouveGotMeowxy** [comment]
am I entering it right? Does it need the http(s)?

I just tried with "one.google.com" and it's also not working in a new private browser.

![image](https://github.com/wallabag/wallabagger/assets/39491297/d8bbd095-eb5b-474e-9f53-1181526da1a6)

**Comment by YouveGotMeowxy** [comment]
It does seem to work when I toggle the "by default" switch though, so maybe I'll just keep that toggled on. :)
